src/QueryBuilder.php

```diff
@@ -187,6 +196,105 @@ public function whereStartsWith($column, string $value, $boolean = 'and', $not =
         return $this;
     }
 
+    /**
+     * Add a script query
+     *
+     * @param  string  $script
+     * @param  array   $options
+     * @param  string  $boolean
+     * @return self
+     */
+    public function whereScript(string $script, array $options = [], $boolean = 'and'): self
```

Comment:
Have you got a link so I can read more about what this "script" type is?

Reply:
Maybe this is just for this example, but wouldn't the booking ID be in the document?

Reply:
Very cool!